src/mame/drivers/vsmjtria.cpp

```diff
@@ -438,10 +481,39 @@ static GFXDECODE_START( gfx_vsmjtria_sub )
 	GFXDECODE_ENTRY( "subgfx", 0, charlayout, 0, 32 )
 GFXDECODE_END
 
+void vsmjtria_state::machine_start()
+{
+	m_keyboard_cmd[0] = m_keyboard_cmd[1] = 0;
+
+	save_item(NAME(m_keyboard_cmd));
+}
+
+void vsmjtria_state::machine_reset()
+{
+	if (m_sw2->read() & 1)
+	{
+		// TODO: there's no setter in nvram_device that directly flushes contents for this case scenario
+		// we currently workaround by directly inject to space address
+		const u16 nvram_size = 0x800;
```

Comment:
I definitely dislike this.
What's the best approach to handle this out, make it a `reset_nvram_contents` public setter in `nvram_device`? Alternatively I may call a `->set_base` pointing to fake filled areas, I'm not convinced either about this.